### ISSUE TITLE
500 Error nuxt instance unavailable - when trying to load client

### ISSUE THREAD

**Issue by emkman** [nuxt-modules/supabase#139]
### Version
@nuxtjs/supabase: 0.3.0
nuxt: 3.0.0

### Steps to reproduce
install `@nuxtjs/supabase`
add `const client = useSupabaseClient();` to top of .vue file
`npm run dev`

### What is Expected?
Page expected to laod

### What is actually happening?
```
Error: nuxt instance unavailable                                                                                                                                                                     17:38:32
    at Module.useNuxtApp (/Users/emkmam/dev/formsmarter/node_modules/nuxt/dist/app/nuxt.mjs:165:13)
    at Module.useSupabaseClient (/Users/emkmam/dev/formsmarter/node_modules/@nuxtjs/supabase/dist/runtime/composables/useSupabaseClient.mjs:10:41)

```

**Comment by emkman** [comment]
I see now this was from trying to use the composable in a SFC with the options API. Is this not supported at the time?

**Comment by salvinoto** [comment]
Im having this exact issue, except all im doing is adding Supabase as a module and I instantly get 500 error. 
`500
Invalid URL

at URL.onParseError (node:internal/url:565:9)
at new URL (node:internal/url:645:5)
at new SupabaseClient (C:\Users\User\Documents\Playground\Nuxt\supabase-nuxt-demo\node_modules\@supabase\supabase-js\dist\module\SupabaseClient.js:77:41)
at Module.createClient (C:\Users\User\Documents\Playground\Nuxt\supabase-nuxt-demo\node_modules\@supabase\supabase-js\dist\module\index.js:20:12)
at Module.useSupabaseAuthClient (C:\Users\User\Documents\Playground\Nuxt\supabase-nuxt-demo\node_modules\@nuxtjs\supabase\dist\runtime\composables\useSupabaseAuthClient.mjs:16:57)
at C:\Users\User\Documents\Playground\Nuxt\supabase-nuxt-demo\node_modules\@nuxtjs\supabase\dist\runtime\plugins\supabase.server.mjs:14:44
at fn (C:\Users\User\Documents\Playground\Nuxt\supabase-nuxt-demo\node_modules\nuxt\dist\app\nuxt.mjs:151:27)
at Object.callAsync (/C:/Users/User/Documents/Playground/Nuxt/supabase-nuxt-demo/node_modules/unctx/dist/index.mjs:49:19)
at callWithNuxt (C:\Users\User\Documents\Playground\Nuxt\supabase-nuxt-demo\node_modules\nuxt\dist\app\nuxt.mjs:153:23)`

**Comment by lancecmarks** [comment]
I am also having this issue.  

Reproduce steps.

Follow Nuxt 3 starter.  'npx nuxi init <project-name>'

Follow nuxt-modules/supabase starter.

'export default defineNuxtConfig({  
    modules: ['@nuxtjs/supabase']
})'

Add SUPABASE_URL and SUPABASE_KEY to Vercel or Netlify environments.

500 Invalid URL

**Comment by GillyRabutTsurwa** [comment]
I am experiencing this exact same issue as well. It breaks whenever I try to instantiate the Supabase client

**Comment by LiamAshdown** [comment]
I get the same problem aswell.

**Comment by GillyRabutTsurwa** [comment]
I have figured out the issue on my part:
When my code was nested under a normal script tag, `<script>.....</script>`, it did **not** work. But when I nested it under the _script setup_ tag, `<script setup>.....</script>` it now works fine with no issues.
To summarise, using the Nuxt Supabase client only works under the script setup tag.
Verify your script tag and change it if necessary. Hopefully this works for you.
